documentation/concepts.md

```diff
@@ -194,6 +251,8 @@ the brokers must be running.
 
 Controller is also referred to as "metadata leader" in some contexts.
 
+[@taylor - guidance is needed on the number of deployed constrollers and possibly the on where they are deployed]
```

Comment:
There is only a single controller (i.e. leader) at a given time which is elected by the Liftbridge cluster. Guidance on cluster size depends, but one important point here is that, currently, all servers in the cluster participate in the Raft consensus group. This has implications on the scalability of the cluster control plane, which there is an issue open to address [here](https://github.com/liftbridge-io/liftbridge/issues/41).

General advice is to run an odd number of servers in the cluster, e.g. 3 or 5, depending on scaling needs. Ideally, cluster members are run in different availability zones or racks for improved fault-tolerance.

Reply:
Excellent! Perhaps you could include this. I am sure user would like to see exactly this, especially coming from you.